openedx/core/djangoapps/course_groups/models.py

```diff
@@ -36,6 +36,11 @@ class Meta:
     GROUP_TYPE_CHOICES = ((COHORT, 'Cohort'),)
     group_type = models.CharField(max_length=20, choices=GROUP_TYPE_CHOICES)
 
+    RANDOM = 'random'
+    MANUAL = 'manual'
+    ASSIGNMENT_TYPE_CHOICES = ((RANDOM, 'Random'), (MANUAL, 'Manual'),)
+    assignment_type = models.CharField(max_length=20, choices=ASSIGNMENT_TYPE_CHOICES, default=MANUAL)
```

Comment:
Definitely a good thing for @ormsbee to weigh in on. It's not really my area of expertise, but @jimabramson what about a second table that just links the groups to the their cohort type (to keep cohort-specific information out of CourseUserGroup)?